pkg/snapstore/abs_snapstore.go

```diff
@@ -48,12 +48,12 @@ type ABSSnapStore struct {
 }
 
 // NewABSSnapStore create new ABSSnapStore from shared configuration with specified bucket
-func NewABSSnapStore(container, prefix, tempDir string, maxParallelChunkUploads uint) (*ABSSnapStore, error) {
-	storageAccount, err := GetEnvVarOrError(absStorageAccount)
-	if err != nil {
-		return nil, err
+func NewABSSnapStore(config *brtypes.SnapstoreConfig) (*ABSSnapStore, error) {
+	prefixString := ""
+	if config.IsSource {
+		prefixString = sourcePrefixString
 	}
```

Comment:
This calculation of the prefix string is repeated several times. I would extract it in an utility function (in `utils.go`), e.g. `getPrefixString(config.IsSource)` and use it directly instead of assigning it to a variable, e.g. `getCredentials(getPrefixString(config.IsSource))`